### PR TITLE
Portfolio: setup and mobile version skeleton - PR

### PR DESCRIPTION
**Pull request by WalterOkumu**
- Created HTML structure inclusive of two sections, that is nav and headline
- Added Font Awesome Icons as the social icon links
- Added hover effect on the social icons
- Completed CSS to closely match figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css" />
+  <script src="https://kit.fontawesome.com/0728e06898.js" crossorigin="anonymous"></script>
+  <title>Microverse Portfolio Website</title>
+</head>
+<body>
+  <div class="navigation">
```
Comment (ShahierNashaat) [inline index.html:12]: - [ ] Kindly be noticed that the container of the section is a `header` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <nav class="nav">
+      <div class="logo">
+        My Logo
+      </div>
```
Comment (ShahierNashaat) [inline index.html:16]: - [ ] Kindly be noticed that the logo is on the left and has a link (`a` tag).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    </nav>
+  </div>
+  <div class="headline">
+    <div class="headline-intro">
+      I am Okumu <br> Glad to see you!
+    </div>
+    <p class="headline-body">
+      I am a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+    <p class="connect">LET US CONNECT</p>
```
Comment (ShahierNashaat) [inline index.html:29]: - [ ] Kindly be noticed that the button `LET US CONNECT` is a link `a` or `button` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <div class="social-icons">
+      <i class="fa-brands fa-twitter twitter-icon"></i>
+      <i class="fa-brands fa-linkedin-in linkedin-icon"></i>
+      <i class="fa-brands fa-medium medium-icon"></i>
+      <i class="fa-brands fa-github github-icon"></i>
+      <i class="fa-brands fa-angellist angellist-icon"></i>
+    </div>
```
Comment (ShahierNashaat) [inline index.html:36]: - [ ] Kindly be noticed that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  font-family: Poppins, sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
```
Comment (ShahierNashaat) [inline css/styles.css:8]: - [ ] Kindly be noticed that you should follow the Figma design in your portfolio and add the proper background image.

**Comment by WalterOkumu** [comment]
Corrections have been adopted

**Pull request closed without merging**